0.neurovault_images.py

```diff
@@ -11,14 +13,24 @@
 import pandas
 import os
 import sys
+import nibabel
+
+try:
+    __IPYTHON__
+    base='/Users/poldrack/data_unsynced/forward-modeling-cognitive-concepts'
```

Comment:
I will remove this, was using for debugging 

Sent from my iPhone

> On Aug 14, 2016, at 5:02 PM, Vanessa Sochat <notifications@github.com> wrote:
> 
> In 0.neurovault_images.py:
> 
> > @@ -11,14 +13,24 @@
> >  import pandas
> >  import os
> >  import sys
> > +import nibabel
> > +
> > +try:
> > +    __IPYTHON__
> > +    base='/Users/poldrack/data_unsynced/forward-modeling-cognitive-concepts'
> Do you want to hard code your system path into the script?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>